lib/internal/test_runner/utils.js

```diff
@@ -128,10 +129,17 @@ function tryBuiltinReporter(name) {
   return require(builtinPath);
 }
 
-async function getReportersMap(reporters, destinations, rootTest) {
+function colorizeTestFiles(rootTest) {
+  const { reporters, destinations } = parseCommandLine();
+  ArrayPrototypeForEach(reporters, (_, index) => {
+    const destination = kBuiltinDestinations.get(destinations[index]) ?? createWriteStream(destinations[index]);
```

Comment:
you should not create a new writeStream here. one is already created within `getReportersMap`